04-single-hop-6lowpan-icmp/README.md

```diff
@@ -1,4 +1,19 @@
-## Goal: Check single-hop ICMP connectivity over IPv6
+Summary
+=======
+
+These tests perform single-hop 6LoWPAN ping tests between nodes in a number of scenarios.
+
+| Task no | Description                                                       | Automated? | Ease of automation |
+|---------|-------------------------------------------------------------------|------------|--------------------|
+| 1       | ICMPv6 link local echo on IoT-Lab M3                              | No         | Hard               |
```

Comment:
There is automation for 1-4 and 7 in #79 so I'd say the automation is there and it is not hard ;-)